src/Telegram.Bot/Exceptions/DefaultExceptionParser.cs

```diff
@@ -93,5 +97,33 @@ private static string TryTruncateErrorDescription(string message, string descrip
                 message = message.Substring(description.Length);
             return message;
         }
+
+        /// <inheritdoc cref="IExceptionParser"/>
+        public async Task<Exception> ParseResponseAsync(HttpResponseMessage httpResponse)
+        {
+            Exception exception;
+
+            string responseJson = await httpResponse.Content.ReadAsStringAsync()
+                .ConfigureAwait(false);
+
+            var apiResponse = JsonConvert.DeserializeObject<JObject>(responseJson);
```

Comment:
Do I need to deserialize response in each custom Exception Handler? How is that better then processing already deserialized data in current implementation? At lest this gives an overhead of having dependency on `Newtonsoft.Json` in extension.